tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -965,6 +965,88 @@ describe(__filename, () => {
     expect(paginator).toHaveProp('pathname', `/user/${user.username}/`);
   });
 
+  it('renders a "description" meta tag when user is a developer', () => {
+    const displayName = 'John Doe';
+    const { store } = dispatchSignInActions({
+      userProps: defaultUserProps({
+        display_name: displayName,
+        is_addon_developer: true,
+        is_artist: false,
+      }),
+    });
+    const root = renderUserProfile({ store });
+
+    expect(root.find('meta[name="description"]')).toHaveLength(1);
+    expect(root.find('meta[name="description"]').prop('content')).toMatch(
+      new RegExp(`The profile of ${displayName}, an extension author`),
+    );
+    expect(root.find('meta[name="description"]').prop('content')).toMatch(
+      new RegExp(`by ${displayName}`),
+    );
+  });
+
+  it('renders a "description" meta tag when user is an artist', () => {
+    const displayName = 'John Doe';
+    const { store } = dispatchSignInActions({
+      userProps: defaultUserProps({
+        display_name: displayName,
+        is_addon_developer: false,
+        is_artist: true,
+      }),
+    });
+    const root = renderUserProfile({ store });
+
+    expect(root.find('meta[name="description"]')).toHaveLength(1);
+    expect(root.find('meta[name="description"]').prop('content')).toMatch(
+      new RegExp(`The profile of ${displayName}, a theme author`),
+    );
+    expect(root.find('meta[name="description"]').prop('content')).toMatch(
+      new RegExp(`by ${displayName}`),
```

Comment:
same here...

Reply:
yep, to make sure the i18n var is correctly replaced.